app/components/paperShow/components/collectionBox.scss

```diff
@@ -0,0 +1,371 @@
+    }
+    &__paper {
+      display: inline-block;
+      vertical-align: top;
+      width: $collection-view-paper-width;
+      margin-right: 16px;
+      &__title {
+        width: 100%;
+        font-size: 14px;
+        font-weight: 500;
+        line-height: 1.4;
+        margin-bottom: 2px;
+        color: $navy0;
+        transition: color ease 0.2s;
+      }
+      &__journal_authors {
+        width: 100%;
+        color: $gray5;
+        font-size: 12px;
+      }
+    }
+    &__memo {
+      display: inline-block;
+      vertical-align: top;
+      width: $collection-view-memo-width;
+      &__content {
+        font-size: 14px;
+        line-height: 1.4;
+        color: black1;
+      }
+    }
+  }
+
+  .action_notification {
+    position: relative;
+    margin-bottom: 8px;
+    &__save {
+      padding: 8px 20px;
+      background-color: $black0;
+      border-radius: 9999px;
+      box-shadow: 2px 8px 8px 0px rgba(0,0,0, 0.25);
+      opacity: 0;
+      position: absolute;
+      right: 0;
+      bottom: 8px;
+      transform: translateX(100%);
+      transition: 
+        opacity ease 0.4s,
+        transform ease 0.4s;
+      &.show {
+        opacity: 1;
+        transform: translateX(0);
+      }
+      span, a {
+        color: $gray3;
+      }
+      span {
+        margin-right: 4px;
+      }
+    }
+  }
+  .action_list {
+    ul li {
+      display: inline-block;
+      position: relative;
+      z-index: 1;
+      box-shadow: 2px 2px 24px 0px rgba(0, 0, 0, 0.25);
+      background: none;
+      &:not(:last-child) {
+        margin-right: 8px;
+      }
+      button {
+        border-radius: 2px;
+        padding: 12px 16px;
+        line-height: 22px;
+        outline: none;
+        cursor: pointer;
+        transition: 
+          color ease 0.2s,
+          background-color ease 0.2s;
+        &:hover {
+          background: $main_blue1;
+          color: $gray0;
+        }
+      }
+    }
+    .collection_view {
+      position: relative;
+      z-index: 0;
+      display: none;
+      width: $collection-view-width;
+      margin-left: 46px;
+      &.show {
+        opacity: 1;
+        display: block;
+      }
+      &__wrapper {
+        position: absolute;
+        padding: 24px;
+        background-color: $gray10;
+        box-shadow: 2px 8px 8px 0px rgba(0, 0, 0, 0.25);
+        bottom: 8px;
+      }
+      .close_button {
+        position: absolute;
+        right: 24px;
+        top: 24px;
+        background: none;
+        border: none;
+        outline: none;
+        cursor: pointer;
+        margin-top: 8px;
+        svg {
+          vertical-align: bottom;
+        }
+      }
+      &__title {
+        font-size: 16px;
+        width: 100%;
+        padding-bottom: 8px;
+        margin: 0;
+        border-bottom: 1px solid #1e2a35;
+        &__link {
+          color: $black0;
+        }
+      }
+      &__list {
+        margin-top: 16px;
+        max-height: 324px;
+        white-space: initial;
```

Comment:
I'm curious what is the intent of initial.
you can find other options from [here](https://developer.mozilla.org/ko/docs/Web/CSS/white-space)